zzz_generated/app-common/clusters/AccessControl/BUILD.gn

```diff
@@ -0,0 +1,35 @@
+# DO NOT EDIT MANUALLY - Generated file
+#
+# Cluster generated code for constants and metadata based on src/controller/data_model/controller-clusters.matter
+import("//build_overrides/chip.gni")
+
+source_set("ids") {
+  sources = [
+    "AttributeIds.h",
+    "ClusterId.h",
+    "CommandIds.h",
+    "EventIds.h",
+    "Ids.h",
+  ]
+  deps = [
+    "${chip_root}/src/app/common:global-ids",
+    "${chip_root}/src/app/data-model-provider:metadata",
```

Comment:
Why do ids depend on provider metadata?  I wouldn't expect them to...

Reply:
This was probably a copy&paste. Removed dependency.